cypress/support/fragments/requests/requests.js

```diff
@@ -299,10 +301,20 @@ export default {
   },
 
   filterRequestsByTag(tag) {
+    cy.wait(2000);
```

Comment:
why do you need waiter here?

Reply:
@manowarqqq same as above, the method is not stable (even on other tests where it is used). Added it to properly deal with Tag chooser.